tests/hint.test

```diff
@@ -51,6 +51,29 @@ Perhaps:
 
 1 hint
 
+---------------------------------------------------------------------
+RUN tests/use-const.hs
```

Comment:
These tests can be written directly in the `<TEST>` section in hlint.yaml. The `.test` files are for tests that need custom hint files, which can't be written there.

Reply:
Thanks! I didn't notice `<TEST>` section in `hlint.yaml` :sweat_smile: